debug.mk

```diff
@@ -9,6 +9,5 @@ RISCV_CXXFLAGS += -O0
 
 # Enable debug
 RISCV_ASFLAGS += -g
-RISCV_CFLAGS += -g
-RISCV_CXXFLAGS += -g
-
+RISCV_CFLAGS += -g -mno-relax
+RISCV_CXXFLAGS += -g -mno-relax
```

Comment:
Since this change is limited to only with debug build, I will allow this for now. We would not want to do this across the board.